agent/containers/images/workloads/Dockerfile.j2

```diff
@@ -0,0 +1,9 @@
+# {{ distro }} pbench-agent workloads image
+FROM quay.io/pbench/pbench-agent-base-{{ distro }}:latest
+
+# Install all the RPMs required for workloads.
+#
+# FIXME: pbench-linpack should be renamed to just linpack.
```

Comment:
We should not depend on linpack because it is not a real RPM that can be publicly built.

Reply:
Removed in latest commit.